hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -2444,8 +2451,16 @@ public FlushResultImpl flushcache(boolean forceFlushAllStores, boolean writeFlus
       }
 
       try {
-        Collection<HStore> specificStoresToFlush =
+        Collection<HStore> specificStoresToFlush = null;
+        if (!forceFlushAllStores && families != null) {
+          specificStoresToFlush = new ArrayList<>();
+          for (byte[] family : families) {
+            specificStoresToFlush.add(stores.get(family));
+          }
+        }else{
```

Comment:
Be careful here. Make code look like rest of the file. Add space around the 'else'.